src/cljs/org/broadinstitute/firecloud_ui/common/components.cljs

```diff
@@ -449,6 +449,50 @@
    (fn [{:keys [refs this]}]
      (.addEventListener (@refs "filter-field") "search" #(when (= (.-value (.-currentTarget %)) "") (react/call :apply-filter this))))})
 
+(react/defc AutocompleteFilter
+  {:apply-filter
+   (fn [{:keys [props refs]}]
+     ((:on-filter props) (common/get-text refs "autocomplete-filter-field")))
+   :render
+   (fn [{:keys [props]}]
+     (let [{:keys [initial-text placeholder width]} props]
+       [:div {:style {:display "inline-flex" :width width}}
+        (style/create-search-field
+          {:ref "autocomplete-filter-field" :autoSave "true" :results 5 :autofocus "true"
+           :placeholder (or placeholder "Filter") :defaultValue initial-text
+           :style {:flex "1 0 auto" :width width :borderRadius 3 :marginBottom 0}
+           :className "typeahead"})]))
+   :component-did-mount
+   (fn [{:keys [refs props locals this]}]
+     (.addEventListener (@refs "autocomplete-filter-field") "search"
+                        #(when (= (.-value (.-currentTarget %)) "")
```

Comment:
I guess? I was just copying what we do in the normal text filter. Should I fix it there too?